utils/build/build.js

```diff
@@ -288,6 +288,19 @@ steps.push({
   concurrent: true,
 });
 
+// Build/watch playwright-dashboard.
```

Comment:
in theory if we add it as [a package](https://github.com/microsoft/playwright/blob/3ad5c2731a3ccccbd8468ef14a88b1edd45867a4/utils/workspace.js#L137) here it gets transpiled automatically [via Babel](https://github.com/microsoft/playwright/blob/31a5a3877bfef1b825688b8d4d9ad88186775de2/utils/build/build.js#L244) like the other packages. Could be also private.